config/fcos/v1_7_exp/translate.go

```diff
@@ -262,6 +262,12 @@ func (c Config) processBootDevice(config *types.Config, ts *translate.Translatio
 				Name:       "root",
 				WipeVolume: util.BoolToPtr(true),
 			}}
+			// Add Kernel Arguments for CEX key file
+			rendered.KernelArguments = types.KernelArguments{
+				ShouldExist: []types.KernelArgument{
+					types.KernelArgument("rd.luks.key=/etc/luks/cex.key"),
```

Comment:
To be 100% sure, the path will always be the same? no chance it would vary between systems?

Reply:
If I understand correctly, it's hardcoded here: https://github.com/coreos/fedora-coreos-config/pull/2986/files#diff-c526dfd1e153e193423394e0ed643fb22184d4d45167d1737754e8564c3f5becR141

Reply:
https://github.com/coreos/ignition/pull/1820/files#diff-01035e9282707d3e15d0f2f82a9869e11b017833be34e0589dab4c5282b708c2